src/Plugins/BangMessage/Inbound/BangMessage.php

```diff
@@ -7,6 +7,18 @@
 
 class BangMessage extends Message
 {
+    /**
+     * The bang command
+     * e.g. the message "!somersault mouse acrobat"
+     * would return "somersault" as the bang command
+     *
+     * @return string
+     */
+    public function bangCommand()
+    {
+        return head(explode(' ', $this->text()));
+    }
```

Comment:
Great idea ;)
We might want to make a bangText/bangParams/bang<something that makes sense> too, so get the text after the `!command`?